content/en/docs/reference/using-api/cel.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CEL in Kubernetes
```

Comment:
nit: Spell out CEL ? What is it and can it be applied to other systems?

Reply:
Thanks. I've replaced "CEL" with "Common Expression Language" here.

This document links to official CEL documentation but tries to mostly avoid duplicating that information. The Language Overview is maybe a bit redundant, but is contextualized enough that I hope it is justified. The rest of the content is specific to how CEL integrates with Kubernetes.